### ISSUE TITLE
[BUG] so uhh, fastflags arent applying 

### ISSUE THREAD

**Issue by conesaresocool** [bloxstraplabs/bloxstrap#3437]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [ ] I did not answer truthfully to all the above checkboxes.

### Bloxstrap Version

v2.8.0

### What problem did you encounter?

so basically the fastflags arent working for me, i checked the wiki and found some %localappdata% thing, but bloxstrap didnt appear when i tried doing that
![Screenshot 2024-10-24 164254](https://github.com/user-attachments/assets/4f3acec2-3238-4f94-aff2-b8aad45e7a8f)
![Screenshot 2024-10-24 164325](https://github.com/user-attachments/assets/df9b0852-203b-47f8-8347-756b947b2d4c)

### Bloxstrap Log

```text
N/A
```

**Comment by conesaresocool** [comment]
![Screenshot 2024-10-24 225318](https://github.com/user-attachments/assets/1d1c8ce6-be85-45b9-ba5d-be9efbbc218a)
also, this is the only thing it can find???

**Comment by conesaresocool** [comment]
> ![Screenshot 2024-10-24 225318](https://private-user-images.githubusercontent.com/186155541/379823678-1d1c8ce6-be85-45b9-ba5d-be9efbbc218a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk3ODE5NDAsIm5iZiI6MTcyOTc4MTY0MCwicGF0aCI6Ii8xODYxNTU1NDEvMzc5ODIzNjc4LTFkMWM4Y2U2LWJlODUtNDViOS1iYTVkLWJlOWVmYmJjMjE4YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyNFQxNDU0MDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMzRmMDU0MTA1OWMwMWUyYTA4NGFiM2RlYjc1NzJlYjM0OWQ3ODdmNjRjYjViZDA3YTIzYTI0NzcyZTJmNDQ0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.L55QEO5mHITDznZeLbQQnPGXzG146lWEJBO88crXSi4) also, this is the only thing it can find???

yeah im actuallyu cocnfuduususuuudusused....

**Comment by pizzaboxer** [comment]
How can you tell that they're not working? Are you seeing the Bloxstrap launcher show before Roblox launches?

**Comment by pizzaboxer** [comment]
Fixed in 2.8.1 (assuming so, reopen if not)
